dynamodb-stream-reader/src/test/java/no/unit/nva/elasticsearch/ElasticSearchRestClientTest.java

```diff
@@ -0,0 +1,41 @@
+package no.unit.nva.elasticsearch;
+
+import elasticsearch.ElasticSearchRestClient;
+import nva.commons.utils.Environment;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ElasticSearchRestClientTest {
+
+    @Test
+    @DisplayName("Constructor with HTTPClient parameter")
+    public void testConstructorWithParameter() {
+        HttpClient httpClient = mock(HttpClient.class);
+        Environment environment = mock(Environment.class);
+        ElasticSearchRestClient elasticSearchRestClient = new ElasticSearchRestClient(httpClient, environment);
+        assertNotNull(elasticSearchRestClient);
+    }
+
+    @Test
+    @DisplayName("doSend")
+    public void testDoSend() throws IOException, InterruptedException {
```

Comment:
what does this test actually test? there is no assertion and everything is mocked

Reply:
It doen nothing at all, was made for code coverage - now its gone